### DISCUSSION TITLE
Double GPU controller

### DISCUSSION THREAD

**Discussion by anphetamina** [Rem0o/FanControl.Releases#2040] [Q&A] [answered]
Is it normal to have two GPU controller for a three fan GPU?

**Comment by Fr0stX76** [comment, marked as answer]
Yes, one control 2 fans, the other the third one.

This is by hardware design, nothing the software can do about it.

**Comment by Rem0o** [comment]
Yes. See README.

https://github.com/Rem0o/FanControl.Releases#faq
